src/main/java/core/basesyntax/Writer.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public class Writer {
+    public void createCsvFile(String toFileName) {
+        File file = new File(toFileName);
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create file " + toFileName, e);
+        }
+    }
+
+    public void writeCsvFileWithData(Path pathFile,
+                                     int buyCount,
+                                     int supplyCount,
+                                     int total) {
+        try {
+            Files.write(pathFile, Arrays.asList("supply," + supplyCount,
+                                                "buy," + buyCount,
+                                                "result," + total));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write file " + pathFile, e);
+        }
+    }
```

Comment:
The class methods should be responsible for single tasks. The 'createCsvFile' method is responsible for creating a file, and 'writeCsvFileWithData' is responsible for writing data to a file. Consider splitting the 'writeCsvFileWithData' method into smaller methods that handle the creation of the CSV content and the writing to the file separately.